shared/rakefiles/entrypoint.rake

```diff
@@ -108,6 +108,55 @@ task :display_cluster_info => [:set_vars] do
   puts
 end
 
+desc "Display gpii/universal image SHA and link to GitHub commit that triggered the build"
+task :display_universal_image_info => [:set_vars] do
+  sh "#{@exekube_cmd} sh -c ' \
+    UNIVERSAL_CI_URL=\"https://ci.gpii.net/\";
+    UNIVERSAL_REPO=\"https://github.com/gpii/universal\";
+    RELEASE_JOB_URL=\"$UNIVERSAL_CI_URL/job/docker-gpii-universal-master-release\";
```

Comment:
Why do you think they are needed here?

I can think of only one common case when variable expansion braces are needed – `var="${first_var}${second_var}"`. Well, another case maybe working with arrays or some string manipulations. Otherwise they contribute to nothing, imo.

Reply:
I guess this is a personal taste, but specially in the cases like `"$UNIVERSAL_CI_URL/job/...` I would use them, as it's not very obvious imho when this will be treated as a variable `${UNIVERSAL}` or variable `${UNIVERSAL_CI_URL/job/...}` - depending on which characters exaclty given shell considers a word separators. Adding `{}` make the intention very clear.

Reply:
Using capital case for variables has the same effect on code readability as using braces with low case variables. Both is probably overkill :) @mrtyler @amatas What do you guys think on this topic?

Reply:
> Using capital case for variables has the same effect on code readability as using braces with low case variables.

Not sure I agree with this for humans, but I disagree for computers -- braces have special meaning to the parser whereas capitalized variable names do not.

> Both is probably overkill

Why?

> What do you guys think on this topic?

I think Stepan's comment is nitpicky and I would not have suggested it. However, I understand his motivation and I would make his suggested change if I had received it on one of my PRs.

Longer-term, I think the solution to debates about code style is to automate code style checking and/or fixing (a la `terraform fmt`). I don't think we're ready to deal with that, especially since we have a fair amount of shell scripts mixed with terraform code, which complicates the tooling. (I've mentioned this before, e.g. https://github.com/gpii-ops/gpii-infra/pull/93#discussion_r211825071.)

But anyway, here is what I got from dropping this shell code into `shellcheck`:

```
In sergey.sh line 22:
        UPSTREAM_JOB_NUMBER=$(curl $RELEASE_JOB_URL/$RELEASE_BUILD/api/json 2> /dev/null | jq -r \".actions[] | select (.causes[0].upstreamBuild != null) | .causes[0].upstreamBuild\");
                            ^-- SC1009: The mentioned parser error was in this command expansion.
                                                                                                                ^-- SC1073: Couldn't parse this select loop.
                                                                                                                       ^-- SC1072: Expected lowercase letter. Fix any mentioned problems and try again.
```

Reply:
@mrtyler Thanks for review!

> braces have special meaning to the parser whereas capitalized variable names do not

Yes, I think we all know that here, context of original discussion was limited to affect of braces on code readability, since code works just fine with them or without them.

> Longer-term, I think the solution to debates about code style is to automate code style checking and/or fixing (a la terraform fmt). I don't think we're ready to deal with that

Agree and agree.

> But anyway, here is what I got from dropping this shell code into shellcheck

Cool, let me address some of those warnings!